tests/Doctrine/Tests/ORM/Functional/Ticket/DDC2917Test.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Tests\ORM\Functional\Ticket;
+
+use Doctrine\ORM\Mapping\Column;
+use Doctrine\ORM\Mapping\DiscriminatorColumn;
+use Doctrine\ORM\Mapping\DiscriminatorMap;
+use Doctrine\ORM\Mapping\Entity;
+use Doctrine\ORM\Mapping\GeneratedValue;
+use Doctrine\ORM\Mapping\Id;
+use Doctrine\ORM\Mapping\InheritanceType;
+use Doctrine\Tests\OrmFunctionalTestCase;
+
+use Throwable;
+use function strtolower;
+
+class DDC2917Test extends OrmFunctionalTestCase
+{
+    protected function setUp(): void
+    {
+        parent::setUp();
+        try {
+                $this->_schemaTool->createSchema(
```

Comment:
I dont believe we need the tables created to test just the DQL generated.

DQL generation tests are also often in a spefic testcase.

I believe this still needs to scope the twst for Pgsql or do we want to add tis behavior to all vendors?

Reply:
> I believe this still needs to scope the twst for Pgsql or do we want to add tis behavior to all vendors?

This works for all vendors, so I didn't scope the test as well. 
The code on the other hand can be scoped, but this requires either a specific strategy which would be a little bit an overkill, either something like `if ($this->platform instanceof PostgreSqlPlatform)` which seem idiosyncratic.
I thought that since this behavior doesn't break anything in other databases, let it be for all vendors.
What do you think?

Reply:
> DQL generation tests are also often in a specific testcase.

Can you explain what do you mean here? I've found only 3 abstract test cases to use.

Reply:
@beberlei can we close this one?